tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/OrderDescriptionControllerTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NHSD.BuyingCatalogue.Ordering.Api.Controllers;
+using NHSD.BuyingCatalogue.Ordering.Api.Models;
+using NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Builders;
+using NHSD.BuyingCatalogue.Ordering.Application.Persistence;
+using NHSD.BuyingCatalogue.Ordering.Domain;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Controllers
+{
+    [TestFixture]
+    internal sealed class OrderDescriptionControllerTests
+    {
+        [Test]
+        public void Constructor_NullRepository_Throws()
+        {
+            Assert.Throws<ArgumentNullException>(() =>
+            {
+                var _ = new OrderDescriptionController(null);
+            });
+        }
+
+        [Test]
+        public async Task Get_OrderIdDoesNotExist_ReturnsNotFound()
+        {
+            var context = OrderDescriptionTestContext.Setup();
+
+            using var controller = context.OrderDescriptionController;
+
+            var response = await controller.GetAsync("INVALID");
+
+            response.Should().BeEquivalentTo(new NotFoundResult());
+        }
+
+        [Test]
+        public async Task Get_OrderIdExists_ReturnsTheOrdersDescription()
+        {
+            var orderId = "C0000014-01";
+
+            var testData = CreateOrderDescriptionTestData(orderId, "Test Description");
+
+            var context = OrderDescriptionTestContext.Setup();
+            context.Order = testData.order;

```

Comment:
I find the lowercase `order` a little strange, but have the opposite when creating the tuple. Could be one for the coding standards meeting as I am not sure what to suggest & it is very minor.

Reply:
Lets discuss it on Friday, as tbh I'm not sure what the best way to do it is. If you want me to change it just say 😄